OpenRA.Mods.Common/Traits/Player/PlaceBuilding.cs

```diff
@@ -117,7 +117,7 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 					if (string.IsNullOrEmpty(segmentType))
 						segmentType = actorInfo.Name;
 
-					foreach (var t in BuildingUtils.GetLineBuildCells(w, targetLocation, actorInfo, buildingInfo))
+					foreach (var t in BuildingUtils.GetLineBuildCells(w, targetLocation, actorInfo, buildingInfo, queue.Actor.Owner))
```

Comment:
Small nitpick: to make it more consistent with the rest of the code we should use `order.Player` here.

Reply:
Fixed.